src/extensions/web-vitals/index.ts

```diff
@@ -22,9 +24,30 @@ export class WebVitalsAutocapture {
 
     constructor(private readonly instance: PostHog) {
         this._enabledServerSide = !!this.instance.persistence?.props[WEB_VITALS_ENABLED_SERVER_SIDE]
+
         this.startIfEnabled()
     }
 
+    public get sampleRate(): number | undefined {
```

Comment:
i'm wondering about taking this out

we have sampling in recording and it makes sense to have that separate since you need to sample by session not by event

but once we add it to web vitals the obvious question from someone is "why not autocapture or pageleave", so I think the feature should be global sampling to keep config from exploding